open_learning_ai_tutor/Intermediary.py

```diff
@@ -27,7 +28,7 @@ def __init__(
             else assessor
         )
         self.intentSelector = (
-            IntentSelector.IntentSelector(intent_history=intent_history)
+            intent_selector.IntentSelector(intent_history=intent_history)
```

Comment:
where is intent_selector defined?

Reply:
You're right - this is broken but also this part of the Intermediary code isn't used. I'm just going to delete it